skillmodels/tests/regression/test_model_one_stage_anchoring.json

```diff
@@ -97,6 +97,6 @@
         "center": false,
         "use_controls": true,
         "use_constant": true,
-        "free_loadings": true
+        "free_loading": true
```

Comment:
This should also be kept an "free_loadings".

Reply:
done